odalic/src/main/java/cz/cuni/mff/xrg/odalic/input/ml/CsvDatasetFileReader.java

```diff
@@ -0,0 +1,101 @@
+package cz.cuni.mff.xrg.odalic.input.ml;
```

Comment:
The best would be to use, for reading of CSV files, the reader which is already there, I mean the current reader Odalic is using - or is there some issue with that?

Reply:
I removed the custom `CsvDatasetFileReader` class and use the `DefaultCsvInputParser` used by odalic. The `ParsingResult` is then converted to the correct format expected by ML Classifier using new `InputValueFactory` class.